src/operations/portal/queries/searchGuardianDirectory.graphql

```diff
@@ -0,0 +1,12 @@
+query SearchGuardianDirectory($search: String!) {
+  searchGuardianDirectory(search: $search) {
+    DOD_ID
```

Comment:
I'm wondering if we need `DOD_ID` here? It's a bit of a hidden feature, but this allows one to search by DoD ID. Not sure if we want that or not.

Reply:
Note: We brought this up with Jing and he thought it was a good hidden feature, so we're leaving this in.